From 316cadc48f61d25551437dda270e35d4c9e5d935 Mon Sep 17 00:00:00 2001
From: Baji Patthan <bpatth@codeaurora.org>
Date: Tue, 5 May 2015 19:37:57 -0700
Subject: [PATCH 13/22] defconfig: msm8916: enable ADV7533 driver

Enable the DBA infrastructure and the ADV7533 bridge driver
for the 8x16 platforms.

Change-Id: I26b59d43d9cd744fff0b9e91cbc178ee8d269167
Signed-off-by: Baji Patthan <bpatth@codeaurora.org>
---
 arch/arm/configs/msm8916-perf_defconfig | 1 +
 arch/arm/configs/msm8916_defconfig      | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/configs/msm8916-perf_defconfig b/arch/arm/configs/msm8916-perf_defconfig
index 517951f..552af51 100644
--- a/arch/arm/configs/msm8916-perf_defconfig
+++ b/arch/arm/configs/msm8916-perf_defconfig
@@ -348,6 +348,7 @@ CONFIG_KGSL_PER_PROCESS_PAGE_TABLE=y
 CONFIG_FB=y
 CONFIG_FB_VIRTUAL=y
 CONFIG_FB_MSM=y
+CONFIG_ADV7533=y
 # CONFIG_FB_MSM_BACKLIGHT is not set
 CONFIG_FB_MSM_MDSS=y
 CONFIG_FB_MSM_MDSS_WRITEBACK=y
diff --git a/arch/arm/configs/msm8916_defconfig b/arch/arm/configs/msm8916_defconfig
index 9b36a70..4724e7f 100755
--- a/arch/arm/configs/msm8916_defconfig
+++ b/arch/arm/configs/msm8916_defconfig
@@ -353,6 +353,7 @@ CONFIG_KGSL_PER_PROCESS_PAGE_TABLE=y
 CONFIG_FB=y
 CONFIG_FB_VIRTUAL=y
 CONFIG_FB_MSM=y
+CONFIG_ADV7533=y
 # CONFIG_FB_MSM_BACKLIGHT is not set
 CONFIG_FB_MSM_MDSS=y
 CONFIG_FB_MSM_MDSS_WRITEBACK=y
-- 
1.8.2.1

